nUpdate.WPFUserInterface/ViewModel/ChangelogVm.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Windows.Forms;
+using System.Windows.Input;
+using System.Windows.Threading;
+using nUpdate.Internal.Core;
+using nUpdate.Internal.Core.Localization;
+using nUpdate.Internal.Core.Operations;
+using nUpdate.ServiceInterfaces;
+using nUpdate.Updating;
+using nUpdate.ViewModel;
+using nUpdate.ViewModel.Interfaces;
+
+// ReSharper disable once CheckNamespace
+namespace nUpdate.WPFUserInterface.ViewModel
+{
+    class ChangelogVm : UpdateUiBaseVm, IDialogViewModel
```

Comment:
Please add modifiers to classes and rename view model classes to "<Name>ViewModel" instead of "NameVm".